cmd/tendermint/commands/init.go

```diff
@@ -33,7 +29,7 @@ func initFiles(cmd *cobra.Command, args []string) {
 				ChainID: cmn.Fmt("test-chain-%v", cmn.RandStr(6)),
 			}
 			genDoc.Validators = []types.GenesisValidator{types.GenesisValidator{
-				PubKey: privValidator.PubKey,
+				PubKey: privValidator.GetPubKey(),
```

Comment:
why not just `PubKey`?

Reply:
because it's already a field.

Once we refactor the PrivValidator more fully (eg. into ID, Signer, LastSignedInfo), we can use `PubKey()`